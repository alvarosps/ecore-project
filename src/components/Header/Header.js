import React from 'react';
import AppBar from '@mui/material/AppBar';
import Toolbar from '@mui/material/Toolbar';
import Typography from '@mui/material/Typography';
import Container from '@mui/material/Container';

const Header = () => {
    return (
        <AppBar position="static" style={{ backgroundColor: '#151c25' }}>
            <Container maxWidth="xl">
                <Toolbar disableGutters>
                    <Typography
                        variant="h6"
                        noWrap
                        component="a"
                        href="/"
                        sx={{
                        mr: 2,
                        display: { xs: 'none', md: 'flex' },
                        fontFamily: 'monospace',
                        fontWeight: 700,
                        letterSpacing: '.3rem',
                        color: 'inherit',
                        textDecoration: 'none',
                        }}
                    >
                        E-CORE Project
                    </Typography>
                    <Typography
                        variant="h5"
                        noWrap
                        component="a"
                        href=""
                        sx={{
                        mr: 2,
                        display: { xs: 'flex', md: 'none' },
                        flexGrow: 1,
                        fontFamily: 'monospace',
                        fontWeight: 700,
                        letterSpacing: '.3rem',
                        color: 'inherit',
                        textDecoration: 'none',
                        }}
                    >
                        E-CORE Project
                    </Typography>
                </Toolbar>
            </Container>
        </AppBar>
    )
}

export default Header;