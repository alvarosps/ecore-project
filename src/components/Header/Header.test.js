import React from 'react';
import renderer from 'react-test-renderer';
import Header from './Header';

describe("Header", () => {
    it('Should render Header', () => {
        const header = renderer.create(<Header />);
        
        expect(header.toJSON()).toMatchSnapshot()
    })
});