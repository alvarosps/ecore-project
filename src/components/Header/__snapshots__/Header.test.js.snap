// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Should render Header 1`] = `
Array [
  <style
    dangerouslySetInnerHTML={
      Object {
        "__html": ".css-349cgn-MuiAppBar-root{display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-flex-direction:column;-ms-flex-direction:column;flex-direction:column;width:100%;box-sizing:border-box;-webkit-flex-shrink:0;-ms-flex-negative:0;flex-shrink:0;position:static;background-color:#1976d2;color:#fff;}",
      }
    }
    data-emotion="css 349cgn-MuiAppBar-root"
  />,
  <style
    dangerouslySetInnerHTML={
      Object {
        "__html": ".css-hip9hq-MuiPaper-root-MuiAppBar-root{background-color:#fff;color:rgba(0, 0, 0, 0.87);-webkit-transition:box-shadow 300ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;transition:box-shadow 300ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;box-shadow:0px 2px 4px -1px rgba(0,0,0,0.2),0px 4px 5px 0px rgba(0,0,0,0.14),0px 1px 10px 0px rgba(0,0,0,0.12);display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-flex-direction:column;-ms-flex-direction:column;flex-direction:column;width:100%;box-sizing:border-box;-webkit-flex-shrink:0;-ms-flex-negative:0;flex-shrink:0;position:static;background-color:#1976d2;color:#fff;}",
      }
    }
    data-emotion="css hip9hq-MuiPaper-root-MuiAppBar-root"
  />,
  <header
    className="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionStatic css-hip9hq-MuiPaper-root-MuiAppBar-root"
    style={
      Object {
        "backgroundColor": "#151c25",
      }
    }
  >
    <style
      dangerouslySetInnerHTML={
        Object {
          "__html": ".css-19r6kue-MuiContainer-root{width:100%;margin-left:auto;box-sizing:border-box;margin-right:auto;display:block;padding-left:16px;padding-right:16px;}@media (min-width:600px){.css-19r6kue-MuiContainer-root{padding-left:24px;padding-right:24px;}}@media (min-width:1536px){.css-19r6kue-MuiContainer-root{max-width:1536px;}}",
        }
      }
      data-emotion="css 19r6kue-MuiContainer-root"
    />
    <div
      className="MuiContainer-root MuiContainer-maxWidthXl css-19r6kue-MuiContainer-root"
    >
      <style
        dangerouslySetInnerHTML={
          Object {
            "__html": ".css-r6ewbb-MuiToolbar-root{position:relative;display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-align-items:center;-webkit-box-align:center;-ms-flex-align:center;align-items:center;min-height:56px;}@media (min-width:0px){@media (orientation: landscape){.css-r6ewbb-MuiToolbar-root{min-height:48px;}}}@media (min-width:600px){.css-r6ewbb-MuiToolbar-root{min-height:64px;}}",
          }
        }
        data-emotion="css r6ewbb-MuiToolbar-root"
      />
      <div
        className="MuiToolbar-root MuiToolbar-regular css-r6ewbb-MuiToolbar-root"
      >
        <style
          dangerouslySetInnerHTML={
            Object {
              "__html": ".css-m5x0z-MuiTypography-root{margin:0;font-family:\\"Roboto\\",\\"Helvetica\\",\\"Arial\\",sans-serif;font-weight:500;font-size:1.25rem;line-height:1.6;letter-spacing:0.0075em;overflow:hidden;text-overflow:ellipsis;white-space:nowrap;color:inherit;margin-right:16px;font-family:monospace;font-weight:700;letter-spacing:.3rem;-webkit-text-decoration:none;text-decoration:none;}@media (min-width:0px){.css-m5x0z-MuiTypography-root{display:none;}}@media (min-width:900px){.css-m5x0z-MuiTypography-root{display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;}}",
            }
          }
          data-emotion="css m5x0z-MuiTypography-root"
        />
        <a
          className="MuiTypography-root MuiTypography-h6 MuiTypography-noWrap css-m5x0z-MuiTypography-root"
          href="/"
        >
          E-CORE Project
        </a>
        <style
          dangerouslySetInnerHTML={
            Object {
              "__html": ".css-1lfs2tq-MuiTypography-root{margin:0;font-family:\\"Roboto\\",\\"Helvetica\\",\\"Arial\\",sans-serif;font-weight:400;font-size:1.5rem;line-height:1.334;letter-spacing:0em;overflow:hidden;text-overflow:ellipsis;white-space:nowrap;color:inherit;margin-right:16px;-webkit-box-flex:1;-webkit-flex-grow:1;-ms-flex-positive:1;flex-grow:1;font-family:monospace;font-weight:700;letter-spacing:.3rem;-webkit-text-decoration:none;text-decoration:none;}@media (min-width:0px){.css-1lfs2tq-MuiTypography-root{display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;}}@media (min-width:900px){.css-1lfs2tq-MuiTypography-root{display:none;}}",
            }
          }
          data-emotion="css 1lfs2tq-MuiTypography-root"
        />
        <a
          className="MuiTypography-root MuiTypography-h5 MuiTypography-noWrap css-1lfs2tq-MuiTypography-root"
          href=""
        >
          E-CORE Project
        </a>
      </div>
    </div>
  </header>,
]
`;
